lucene/suggest/src/java/org/apache/lucene/search/suggest/document/TopSuggestDocsCollector.java

```diff
@@ -201,4 +205,27 @@ public void collect(int doc) throws IOException {
   public ScoreMode scoreMode() {
     return ScoreMode.COMPLETE;
   }
+
+  /**
+   * returns true if the collector clearly exhausted all possibilities to collect results
+   */
+  boolean isComplete() {
```

Comment:
Is this needed now that we provide the information in the `TopSuggestDocs` ?

Reply:
Not necessary, but since we need the underlying flag and its currently private, I thought its okay to have at least a package private getter. Or does it bloat the class too much? Happy to remove either way, wdyt?